requirements/dev.txt

```diff
@@ -6,97 +6,37 @@
 #
 asn1crypto==0.24.0        # via cryptography
 aspy.yaml==1.1.2          # via pre-commit
-attrdict==2.0.0
-attrs==18.2.0
-awesome-slugify==1.6.5
 bcrypt==3.1.6             # via paramiko
 bleach==3.1.0             # via readme-renderer
-cached-property==1.5.1
-certifi==2018.11.29
-cffi==1.12.0              # via bcrypt, cryptography, pynacl
+cffi==1.12.1              # via bcrypt, cryptography, pynacl
 cfgv==1.4.0               # via pre-commit
-chardet==3.0.4
-click-default-group==1.2
-click==7.0
-colorlog==4.0.2
 cryptography==2.5         # via paramiko
-cytoolz==0.9.0.1
-decorator==4.3.2
-defusedxml==0.5.0
-dill==0.2.9
 docopt==0.6.2             # via ptpython
 docutils==0.14            # via readme-renderer
-et-xmlfile==1.0.1
-eth-abi==1.3.0
-eth-account==0.3.0
-eth-hash[pycryptodome]==0.2.0
-eth-keyfile==0.5.1
-eth-keys==0.2.1
-eth-rlp==0.1.2
-eth-typing==2.0.0
-eth-utils==1.4.1
 fabric3==1.14.post1
 filelock==3.0.10          # via tox
-hexbytes==0.1.0
 identify==1.2.2           # via pre-commit
-idna==2.8
 importlib-metadata==0.8   # via pre-commit
 importlib-resources==1.0.2  # via pre-commit
-ipython-genutils==0.2.0
-jdcal==1.4
 jedi==0.13.2              # via ptpython
-jsonschema==2.6.0
-jupyter-core==4.4.0
-lru-dict==1.1.6
-nbformat==4.4.0
 nodeenv==1.3.3            # via pre-commit
-odfpy==1.4.0
-openpyxl==2.5.14
 paramiko==2.4.2           # via fabric3
-parsimonious==0.8.1
 parso==0.3.4              # via jedi
-pendulum==2.0.4
-pint==0.9
-pip-tools==3.3.1
-plotly==3.5.0
 pluggy==0.8.1             # via tox
-pre-commit==1.14.3
+pre-commit==1.14.4
 prompt-toolkit==2.0.8     # via ptpython
 ptpython==2.0.4
 py==1.7.0                 # via tox
 pyasn1==0.4.5             # via paramiko
 pycparser==2.19           # via cffi
-pycryptodome==3.7.3
 pygments==2.3.1           # via ptpython, readme-renderer
 pynacl==1.3.0             # via paramiko
-python-dateutil==2.7.5
-python-rex==0.4
-pytz==2018.9
-pytzdata==2018.7
-pyyaml==3.13
+pyyaml==3.13              # via aspy.yaml, pre-commit
```

Comment:
Sorry, but pyyaml==3.13 is still there which will lead to the github warning.

Reply:
Shit, I thought that since it was removed from base.txt pyyaml was no longer there. I will fix the version to the one GitHub suggests. Thanks